docs/deploy.rst

```diff
@@ -0,0 +1,229 @@
++------------------------+-------------------------------------------------+
+| ``--cluster-name``     | The CrateDB cluster name to use for deployment. |
++------------------------+-------------------------------------------------+
+| ``--project-id``       | The project ID to use for deployment.           |
++------------------------+-------------------------------------------------+
+| ``--version``          | The CrateDB version to use for deployment.      |
++------------------------+-------------------------------------------------+
+| ``--username``         | The CrateDB username to use for deployment.     |
++------------------------+-------------------------------------------------+
+| ``--password``         | The CrateDB password to use for deployment.     |
++------------------------+-------------------------------------------------+
+
+For this test deployment, the command should look like this:
+
+.. code-block:: console
+
+   sh$ croud clusters deploy \
+       --product-name cratedb.az-gp1 \
+       --tier xxs \
+       --cluster-name my-first-crate-cluster \
+       --project-id 59c7961b-33a2-4478-9e98-2a6b3f54f60e \
+       --version 3.3.0 \
+       --username yourusername \
+       --password "yourpassword" \
+       --region bregenz.a1 \
+       --env prod
+
+If the operation was successful the command will output:
+
+.. code-block:: console
+
+   +--------------------------------------+------------------------+----------+-------------+--------------------------------------+----------+------------------------------------------------------+
+   | id                                   | name                   | numNodes | crateVersion| projectID                            | username | fqdn                                                 |
+   +--------------------------------------+------------------------+----------+-------------+--------------------------------------+----------+------------------------------------------------------+
+   | 8d6a7c3c-61d5-11e9-a639-34e12d2331a1 | my-first-crate-cluster |        1 | 3.3.0       | 59c7961b-33a2-4478-9e98-2a6b3f54f60e | default  | my-first-crate-cluster.bregenz.a1.cratedb.net. |
+   +--------------------------------------+------------------------+----------+-------------+--------------------------------------+----------+------------------------------------------------------+
+  ==> Success: Cluster deployed. It may take a few minutes to complete the changes.
+
+.. CAUTION::
+
+  The cluster deployment will still be in progress and take around 5-10 minutes
+  to complete.
+
+At the prompt, enter the CrateDB Cloud credentials you have set previously in
+this turorial.
+
+You should now see the administration user interface, AdminUI for short, of your
+first deployed CrateDB Cluster.
+
+Your CrateDB cluster is up and running. Congratulations.
+
+Next steps
+==========
+
+You have deployed your first CrateDB cluster.
```

Comment:
this needs developing

perhaps link through to getting started docs for cratedb re querying etc